docs/pages/getting-started.md

```diff
@@ -0,0 +1,39 @@
+# Getting started
+When you have installed all the [prerequisites](/prerequisites.md) you are good to go.
+
+Start by installing the Simple Fable template:
+```
+dotnet new --install "Fable.Template::*"
+```
+Create a directory for your project:
+```
+mkdir FableApp
+cd FableApp
+```
+Scaffold a Fable app from the template:
+```
+dotnet new Fable
```

Comment:
It's also possible to create the directory and start the project there in one step (without `mkdir`) using `dotnet new fable -n FableApp`